packages/release/Cargo.toml

```diff
@@ -38,6 +38,7 @@ libaudit = { path = "../libaudit" }
 libgcc = { path = "../libgcc" }
 libstd-rust = { path = "../libstd-rust" }
 makedumpfile = { path = "../../packages/makedumpfile" }
+netdog = {path = "../netdog" }
```

Comment:
`release` should no longer depend on `wicked` - right now this dependency is forcing it to be built for all variants, but it should only be built for variants that include it.